lib/gitsh/interactive_runner.rb

```diff
@@ -16,6 +18,7 @@ def initialize(opts)
       @history = opts.fetch(:history, History.new(@env, @readline))
       @interpreter = opts.fetch(:interpreter, Interpreter.new(@env))
       @term_info = opts.fetch(:term_info) { TermInfo.instance }
+      @script_runner = opts.fetch(:script_runner) { ScriptRunner.new(env: @env) }
```

Comment:
Hm, why use a block here and for `@term_info` but not for `@interpreter` or `@history`?